components/blitz/resources/omero/Constants.ice

```diff
@@ -214,14 +214,6 @@ module omero {
      **/
     module analysis {
 
-       /** namespaces related to the FLIM analysis. **/
```

Comment:
To follow our own rules, I'd say let's go ahead and deprecate these rather than remove them, and just explicitly mention in the deprecation message that they'll be removed 5.4 and above.